### PR TITLE
ref(ui): Use DropdownMenuV2 for ignore & resolve dropdowns

### PR DESCRIPTION
**Pull request by vuluongj20**
(To be merged into master after https://github.com/getsentry/sentry/pull/31084)

Use DropdownMenuControlV2, which has an updated design, better keyboard accessibility and aria labeling.

**Before:**
Screen Shot 2022-01-26 at 2 31 43 PM
Screen Shot 2022-01-26 at 2 31 50 PM
Screen Shot 2022-01-26 at 2 31 56 PM

**After:**
Screen Shot 2022-01-26 at 2 32 08 PM
Screen Shot 2022-01-26 at 2 32 19 PM
Screen Shot 2022-01-26 at 2 32 24 PM

### CODE REVIEW

File: static/app/components/actions/resolve.tsx [hunk 1]
```diff
@@ -134,61 +132,62 @@ class ResolveActions extends React.Component<Props> {
       ? t('Set up release tracking in order to use this feature.')
       : '';
 
-    const actionLinkProps = {
-      shouldConfirm,
-      message: confirmMessage,
-      confirmLabel,
-      disabled: disabled || !hasRelease,
+    const onActionOrConfirm = onAction => {
+      if (shouldConfirm) {
```
Comment (evanpurkhiser) [inline static/app/components/actions/resolve.tsx:136]: You can get rid of these `if/else` by passing the [`bypass` parameter](https://github.com/getsentry/sentry/blob/9ffb055577cce118fb51042273bc57b90b73329f/static/app/components/confirm.tsx#L133) to `openConfirmModal`.

**Review by evanpurkhiser — APPROVED** [review]
It's less code and I like it

File: static/app/components/actions/resolve.tsx [hunk 2]
```diff
@@ -134,61 +132,62 @@ class ResolveActions extends React.Component<Props> {
...
-        disabled={!projectSlug ? disabled : disableDropdown}
-      >
-        <MenuHeader>{t('Resolved In')}</MenuHeader>
-
-        <MenuItemActionLink
-          {...actionLinkProps}
-          title={t('The next release')}
-          onAction={this.handleNextReleaseResolution}
-        >
-          <Tooltip disabled={hasRelease} title={actionTitle}>
```
Comment (vuluongj20) [inline static/app/components/actions/resolve.tsx:164]: Removing tooltip here in favor of the `details` prop. This just makes the text more visible.

Before:
Screen Shot 2022-01-28 at 12 46 58 PM

After:
Screen Shot 2022-01-28 at 12 46 29 PM

Comment (evanpurkhiser) [reply static/app/components/actions/resolve.tsx:164]: Ahh I LOVE this

File: static/app/components/actions/resolve.tsx (hunk 1 above)
Comment (vuluongj20) [reply static/app/components/actions/resolve.tsx]: Ahh that's much nicer!

**Comment by priscilawebdev** [comment]
did we add the ellipsis after "Another existing release" on purpose? 

image

**Review by priscilawebdev — APPROVED** [review]
Great work 🎉

**Comment by vuluongj20** [comment]
> did we add the ellipsis after "Another existing release" on purpose?
> 
> image

Yepp I added it. Thought it made sense because we require additional user input after selecting this option, and we already have a similar pattern in the Ignore dropdown.

**Pull request merged**